File: Algorithms/BubbleSort/bubbleSort.py
def bubbleSort(list):
    swap = False
    for a in range(len(list) -1):
        if list[a] > list[a+1]:
            swap = True
            list[a], list[a+1] = list[a+1], list[a]

    if not swap:
        return list

    return bubbleSort(list)

File: Algorithms/BubbleSort/bubblesort.js
function bubbleSort(list) {
	let swap = false;
	for (let i = 0; i <= list.length -1; i++) {
		if (list[i] > list[i+1]){
			t = list[i];
			list[i] = list[i+1];
			list[i+1] = t;

			swap = true;

		}
	}

	if (!swap){
		return list;
	}
	
	return bubbleSort(list);
	
}